### ISSUE TITLE
Analysis Location supporting Nested Jars

### ISSUE THREAD

**Issue by JonasKlauke** [soot-oss/SootUp#750]
If a jar contains other Jars the input location should also add them to the view.

For this we might to also change the way how Fully Qualified names are generated. Issue #675 shows an example of  kind of nested classes.

**Comment by JonasKlauke** [comment]
similiar to #759

**Comment by swissiety** [comment]
> similiar to [#759](https://github.com/soot-oss/SootUp/issues/759)

yes but no - its a bit simpler as we dont need to map all the implicit connections as well.
